src/common/Dropdown.js

```diff
@@ -1,15 +1,33 @@
-import React, { useRef } from 'react';
+import React, { useRef, useEffect, useState } from 'react';
 import './Dropdown.scss';
 import useDetectOutsideClick from '../hooks/useDetectOutsideClick';
 import Logout from '../navbar/Logout';
 import dashboardIcon from '../icons/dropdown_dashboard.svg';
 import settingsIcon from '../icons/dropdown_settings.svg';
 
+import useRole from '../hooks/useRole';
+
 function Dropdown({ name, profileURL, onLogoutSuccess }) {
   const dropdownRef = useRef(null);
   const [isActive, setIsActive] = useDetectOutsideClick(dropdownRef, false);
   const onClick = () => setIsActive(!isActive);
 
+  const { role, roleError } = useRole();
+  const [dashboardLink, setDashboardLink] = useState('/');
+
+  useEffect(() => {
+    switch (role) {
```

Comment:
could this be cleaner as a ternary ? operator

Reply:
eslint wont let me do 3 way ternary :(((((((((